app/src/main/index.js

```diff
@@ -436,10 +436,10 @@ ipcMain.on('open-editor-window', (event, opts) => {
   }
 
   editorWindow = new BrowserWindow({
-    width: 768,
-    minWidth: 768,
-    height: 428,
-    minHeight: 428,
+    width: 800,
+    minWidth: 800,
+    height: 430,
+    minHeight: 430,
```

Comment:
The text and size block are stacked as I've added a new button 'APNG'
Old values: 
![screen shot 2017-05-24 at 13 43 01](https://cloud.githubusercontent.com/assets/1215719/26401605/faac1822-4086-11e7-9ef4-e25d3da6046a.png)

New values (800x430) 
![screen shot 2017-05-24 at 13 41 52](https://cloud.githubusercontent.com/assets/1215719/26401632/0d40fbb0-4087-11e7-87b9-681ceefdbb38.png)